specreduce/tests/test_get_reference_file_path.py

```diff
@@ -8,3 +8,12 @@ def test_get_reference_file_path():
     test_path = "extinction/apoextinct.dat"
     p = get_reference_file_path(path=test_path)
     assert p is not None
+
+
+def test_get_pypeit_data_path():
```

Comment:
If your test downloads data, you might wanna mark it with `@pytest.mark.remote_data`.

Reply:
good call. i'll add that.

Reply:
done. i tweaked the CI workflow so that `--remote-data` is only enabled for the one test where coverage is measured. this should hopefully help with the occasional failures due to downloads timing out. no longer parallel tests getting the same files at the same time...